.github/workflows/scorecard.yml

```diff
@@ -19,6 +19,11 @@ jobs:
       id-token: write
 
     steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@v2
+        with:
```

Comment:
interesting that this one does not have the commit hash of the version

Reply:
Yes, I noticed this too. Luckily renovate should auto tag it in a follow up PR.

Reply:
just pinned the sha for it. 

are they also an oss project?